tests/pyccel/test_pyccel.py

```diff
@@ -786,6 +786,17 @@ def test_classes( test_file , language):
     else:
         pyccel_test(test_file, compile_with_pyccel = False, language=language)
 
+#------------------------------------------------------------------------------
+@pytest.mark.parametrize( "test_file", ["scripts/classes/classes_2.py",
```

Comment:
Shouldn't this file be tested for Python and Fortran too? Either by adding Fortran and Python to the languages run here, or by adding this file to the list used by `test_classes`

Reply:
Fixed